Makefile

```diff
@@ -168,6 +169,30 @@ host-agent-binary: $(RELEASE_DIR)
 		-o ./bin/$(notdir $(RELEASE_BINARY))-$(GOOS)-$(GOARCH) $(HOST_AGENT_DIR)
 
 
+##@Release
+
+$(RELEASE_DIR):
+	rm -rf $(RELEASE_DIR)
+	mkdir -p $(RELEASE_DIR)
+
+build-release-artifacts: build-cluster-templates build-infra-yaml build-metadata-yaml build-host-agent-binary
+
+build-cluster-templates: $(RELEASE_DIR) cluster-templates
+	cp $(BYOH_TEMPLATES)/v1beta1/cluster-template.yaml $(RELEASE_DIR)/cluster-template.yaml
+	sed -i -e 1,20d $(RELEASE_DIR)/cluster-template.yaml
+
+IMG=projects.registry.vmware.com/cluster_api_provider_bringyourownhost/cluster-api-byoh-controller:rel-tag
```

Comment:
The rel-tag is the release tag. It should be same as the tag that was pushed that triggered this job.

Reply:
Maybe we can pass the value from the `draft-release.yaml` file?

Reply:
I think this should work.
```
- name: Build Release Artifacts
        run: IMG="projects.registry.vmware.com/cluster_api_provider_bringyourownhost/cluster-api-byoh-controller:${{ env.GITHUB_REF }}" make build-release-artifacts

```